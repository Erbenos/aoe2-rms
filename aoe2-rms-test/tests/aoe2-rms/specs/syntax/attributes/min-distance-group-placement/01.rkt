#lang aoe2-rms
; Give each player two sets of forages and make them avoid each other by 4 tiles, and keep all future objects 4 tiles away.

<OBJECTS-GENERATION>
(create-object 'FORAGE
               (number-of-objects 7)
               (number-of-groups 2)
               (set-tight-grouping)
               (set-place-for-every-player)
               (set-gaia-object-only)
               (min-distance-to-players 8)
               (max-distance-to-players 10)
               (min-distance-group-placement 4))
