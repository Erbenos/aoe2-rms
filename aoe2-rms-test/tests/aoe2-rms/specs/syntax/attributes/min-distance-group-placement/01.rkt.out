<OBJECTS_GENERATION>
create_object FORAGE {
  number_of_objects 7
  number_of_groups 2
  set_tight_grouping
  set_place_for_every_player
  set_gaia_object_only
  min_distance_to_players 8
  max_distance_to_players 10
  min_distance_group_placement 4
}
